lib/meilisearch-rails.rb

```diff
@@ -858,6 +865,17 @@ def ms_attribute_changed?(document, attr_name)
         # We don't know if the attribute has changed, so conservatively assume it has
         true
       end
+
+      def warn_searchable_missing_attributes
+        if (searchables = meilisearch_settings.get_setting(:searchable_attributes)) &&
```

Comment:
It is okay to take some extra new lines to create the variables first. This way it will be easier to understand the logic behind the alert.